src/Workspaces/CSharpTest/CodeGeneration/AddImportsTests.cs

```diff
@@ -1240,7 +1240,7 @@ class C
 }", safe: true, useSymbolAnnotations);
         }
 
-        [Theory(Skip = "https://github.com/dotnet/roslyn/issues/39641"), InlineData(true), InlineData(false)]
```

Comment:
>https://github.com/dotnet/roslyn/issues/39641 [](start = 24, length = 45)

The PR doesn't call out [https://github.com/dotnet/roslyn/issues/39641](https://github.com/dotnet/roslyn/issues/39641) as fixed. Is it?

Reply:
Thanks, meant to update and didn't.